product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/preparing_gcp/index.mdx

```diff
@@ -1,3 +1,112 @@
+    ```
+    The required arguments are:
+
+    | Argument | Description |
+    | -------- | ----------- |
+    | &lt;account-id> | Google Cloud account ID of your BigAnimal deployment. |
+    | &lt;region> | Google Cloud region where your clusters are being deployed. See [Supported regions](../../overview/03a_region_support/#supported-google-cloud-regions) for a list of possible regions. |
+
+    Possible options are:
+
+    | Options | Description |
+    | ------- | ----------- |
+    | `-h` or `--help`| Displays the command help. |
+    | `-i` or `--instance-type` | Google Cloud instance type for the BigAnimal cluster. The help command provides a list of possible VM instance types. Choose the instance type that best suits your application and workload. Choose an instance type in the memory optimized M1, M2, or M3 series for large data sets. Choose from the compute-optimized C2 or C2D series for compute-bound applications. Choose from the accelerator-optimized A2 or A2 ultra for massively parallelized Compute Unified Device Architecture (CUDA) workloads. Choose from the general purpose M5 or M6I series if you don't require memory or compute optimization.|
+    | `-x` or `--cluster-architecture` | Defines the Cluster architecture and can be `single`, `ha`, or `eha`. See [Supported cluster types](/biganimal/release/overview/02_high_availability) for more information.|
+    | `-e` or `--endpoint` | Type of network endpoint for the BigAnimal cluster, either `public` or `private`. See [Cluster networking architecture](../creating_a_cluster/01_cluster_networking) for more information. |
+    | `-r` or `--activate-region` | Specifies region activation if no clusters currently exist in the region. |
+    | `--onboard` | Checks if the user and subscription are correctly configured.
+
+    The behavior of the script defaults to `--onboard` if you provide no other options.
+
+    For example, if you want to deploy a cluster in a Google Cloud account having an ID of `1234-5678-9012`, with an instance type of `n2-standard-8`, in the `us-east1` region, with a `public` endpoint, and with no existing cluster deployed, run the following command:
+
+    ```shell
+    curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/gcp/biganimal-csp-preflight | bash -s \
+    1234-5678-9012 \
+    us-east1 \
+    --instance-type n2-standard-8 \
+    --endpoint public \
+    --activate-region \
+    --onboard\
+    ```
+
+The script displays the following output:
+
+    ```
+    ######################################################
+    Run GCP Preflight Checks with Google Cloud CLI 436.0.0
+
+    ####################################
+    # Checking for enabled GCP APIs... #
+    ####################################
+
+    NAME                                     RESULT
+    ---------------------------------------  ---------
+    autoscaling.googleapis.com               Enabled
+    cloudapis.googleapis.com                 Enabled
```

Comment:
```suggestion
    cloudapis.googleapis.com                 Enabled
    cloudresourcemanager.googleapis.com      Enabled
```

Reply:
I forgot this before, is being added with https://github.com/EnterpriseDB/cloud-utilities/pull/124/files#diff-30c64f35caf3a8d9af1ebc2320b70ec40752f17fb2defbdd6d75be728e83ff87R509